assets/src/components/activities/DeliveryElement.ts

```diff
@@ -73,6 +73,11 @@ export interface DeliveryElementProps<T extends ActivityModelSchema> {
    */
   state: ActivityState;
 
+  /**
```

Comment:
Recently I did some work to cleanup the `DeliveryElement` interface.  It had gotten out of hand, with a whole bunch of top level properties having been added.  I think it makes sense to strive to continue to keep that interface small.  So instead of adding a new top level attr for `page-state` I'd recommend adding `pageState` as an attribute of the top level `activityContext instead.

Reply:
ah yes, good point. thanks